packages/sdk/src/NomadContext.ts

```diff
@@ -173,6 +180,51 @@ export class NomadContext extends MultiProvider<config.Domain> {
     return this.mustGetCore(this.governor.domain);
   }
 
+  /**
+   * Proves and Processes a transaction on the destination chain. This is subsidize and
+   * automatic on non-Ethereum destinations
+   * 
+   * @dev Ensure that a transaction is ready to be processed. You should ensure the following
+   * criteria have been met prior to calling this function:
+   *  1. the tx has been relayed
+   *  2. the `confirmAt` timestamp for the tx is in the past
+   *  3. the user is on the destination chain in their wallet
+   *
+   * @returns The Contract Transaction receipt
+   */
+  async process(origin: string | number, txId: string): Promise<ContractTransaction>{
+    // get s3 proof URL for environment
+    const { environment } = this.conf
+    const s3URL = s3URLs[environment]
+
+    const message = (await NomadMessage.baseFromTransactionHash(
+      this,
+      origin,
+      txId
+    ))[0]
+
+    const originNetwork = this.resolveDomainName(message.origin)
+    const destNetwork = this.resolveDomainName(message.destination)
+    const index = message.leafIndex.toString()
+    const s3Res = await fetch(`${s3URL}${originNetwork}_${index}`)
+    const data = (await s3Res.json()) as any
```

Comment:
You can probably remove any just fine here (and the surrounding curly braces)

Reply:
cast it to the type we actually expect :)